operators/to_mapbox/__init__.py

```diff
@@ -0,0 +1,152 @@
+        DF.update_resource(['Responses'], **{'name': 'responses', 'path': 'responses.csv'}),
+        DF.select_fields(['pk', 'id', 'name', 'description'], resources=['responses']),
+
+        # SERVICES PRE-PROCESSING
+        DF.update_resource(['Services'], **{'name': 'services', 'path': 'services.csv'}),
+        DF.filter_rows(lambda r: r['selected'] is True, resources=['services']),
+        DF.add_field('organization_pk', 'string', resources=['services']),
+        unwind('organizations', 'organization_pk', resources=['services']),
+        DF.add_field('response_pk', 'string', resources=['services']),
+        unwind('responses', 'response_pk', resources=['services']),
+        DF.select_fields(['pk','id', 'selected', 'name', 'organization_pk', 'response_pk'], resources=['services']),
+
+        # ORGANIZATION PRE-PROCESSING
+        DF.update_resource(['Organizations'], **{'name': 'organizations', 'path': 'organizations.csv'}),
+        DF.add_field('branch_pk', 'string', resources=['organizations']),
+        unwind('branches', 'branch_pk', resources=['organizations']),
+        DF.select_fields(['pk','id', 'name', 'branch_pk'], resources=['organizations']),
+
+        # BRANCH PRE-PROCESSING
+        DF.update_resource(['Branches'], **{'name': 'branches', 'path': 'branches.csv'}),
+        DF.add_field('location_pk', 'string', resources=['branches']),
+        unwind('location', 'location_pk', resources=['branches']),
+        DF.add_field('organization_pk', 'string', resources=['branches']),
+        unwind('organization', 'organization_pk', resources=['branches']),
+        DF.select_fields(['pk', 'id', 'name', 'location_pk', 'organization_pk'], resources=['branches']),
+
+        # LOCATION PRE-PROCESSING
+        DF.update_resource(['Locations'], **{'name': 'locations', 'path': 'locations.csv'}),
+        DF.filter_rows(lambda r: any(
+            all(r.get(f) for f in fields)
+            for fields in [('resolved_lat', 'resolved_lon'), ('fixed_lat', 'fixed_lon')]
+        ), resources=['locations']),
+        DF.add_field('lat', 'number', lambda r: r.get('fixed_lat') or r['resolved_lat'], resources=['locations']),
+        DF.add_field('lon', 'number', lambda r: r.get('fixed_lon') or r['resolved_lon'], resources=['locations']),
+        DF.add_field('geometry', 'geopoint', lambda r: [r['lon'], r['lat']], resources=['locations']),
+        DF.add_field('address', 'string', lambda r: r.get('resolved_address') or r['id'], resources=['locations']),
+        DF.select_fields(['pk', 'geometry', 'address'], resources=['locations']),
+
+        # JOIN TO PRODUCE MAPBOX DATA
+        DF.duplicate('services', 'to_mapbox', 'to_mapbox.csv'),
+        DF.rename_fields({'name': 'service_name'}, resources=['to_mapbox']),
+        DF.join(
+            'locations', ['pk'], 'branches', ['location_pk'],
+            fields={'geometry': {'name': 'geometry', 'aggregate': 'last'}, 'address': {'name': 'address', 'aggregate': 'last'}},
```

Comment:
having the same name and aggregate last are the defaults, so you could actually have `dict(geometry=None, address=None)` to achieve the same effect here.
(just fyi, not CR)

Reply:
good to know.